lib/replica/home.dart

```diff
@@ -53,16 +55,38 @@ class _ReplicaHomeScreenState extends State<ReplicaHomeScreen> {
 
   @override
   Widget build(BuildContext context) {
+    print("Home called");
     // We are showing the ReplicaSearchScreen here since we want the bottom tabs to be visible (they are not if it's its own route)
     // <08-23-22, kalli>  Not ideal UX - maybe add a spinner? Not sure
     // <09-07-22, kalli> Update after testing a debug build - this is not very noticeable.
     if (showResults) {
       return ReplicaSearchScreen(
```

Comment:
It looks like we use the same `textEditingController` (with `replicaSearchInput` label) in the ReplicaSearchScreen widget. Is it possible to pass the instance created here when we construct a ReplicaSearchScreen vs. maintaining two separate instances? Like this: https://github.com/getlantern/lantern-client/compare/jigar/replica-fixes...atavism/replica-fixes?expand=1

Reply:
Good Idea, Thanks for making changes.

Reply:
It looks like we using two controllers we facing some issues due Life cycle of the widget. I think we should go the same for now, Also Once users start searching they will anyway goto the next screen and use that onwards.

Let me know what you think?